osu.Game.Rulesets.Diva/Mods/DivaModKey3.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Diva.Mods
+{
+	public class DivaModKey3 : DivaKeyMod
+	{
+		public override int KeyCount => 3;
+		public override string Name => "3 Button";
```

Comment:
should be "Three Button" for consistency

Actually make Button plural on all ModKey as well?